usaspending_api/bulk_download/config/supervisord.conf

```diff
@@ -12,8 +12,23 @@ user=ec2-user
 environment=
     PYTHONPATH="%(ENV_PATH)s:/data-act/backend",
     DATABASE_URL="%(ENV_DATABASE_URL_VAR)s",
-    DOWNLOAD_DATABASE_URL="%(ENV_DOWNLOAD_DATABASE_URL)s"
-command=python3 manage.py download_sqs_worker
+    DOWNLOAD_DATABASE_URL="%(ENV_DOWNLOAD_DATABASE_URL)s",
+    # ==== [Open Telemetry Configuration] ====
+    OTEL_SERVICE_NAME="usaspending-downloader",
+    OTEL_TRACES_EXPORTER="otlp",
+    OTEL_METRICS_EXPORTER="otlp",
+    OTEL_PYTHON_LOG_CORRELATION="true",
+    OTEL_PYTHON_LOG_FORMAT="%(msg)s [span_id=%(otelSpanID)s trace_id=%(otelTraceID)s]",
+    OTEL_PYTHON_LOG_LEVEL="debug",
+    OTEL_PYTHON_LOGGING_AUTO_INSTRUMENTATION_ENABLED="true",
+    OTEL_EXPORTER_OTLP_PROTOCOL="http/protobuf",
+    OTEL_INSTRUMENTATION_HTTP_CAPTURE_HEADERS_SANITIZE_FIELDS=".*session.*,set-cookie",
+    OTEL_EXPORTER_OTLP_TIMEOUT="30000",
+    OTEL_INSTRUMENTATION_HTTP_CAPTURE_HEADERS_SERVER_REQUEST="content-length,content-type,host,origin,referer,ua-is-bot,user-agent,x-forwarded-for,x-requested-with,allow,cache-trace,is-dynamically-rendered,key,strict-transport-security",
+    OTEL_INSTRUMENTATION_HTTP_CAPTURE_HEADERS_SERVER_RESPONSE="content-length,content-type,host,origin,referer,ua-is-bot,user-agent,x-forwarded-for,x-requested-with,allow,cache-trace,is-dynamically-rendered,key,strict-transport-security",
+    GRPC_VERBOSITY="debug",
```

Comment:
Do we want this to be set to `debug` outside of local development instances?

Reply:
Set this to `info`